package.json

```diff
@@ -10,20 +9,33 @@
   "repository": "git@github.com:bence-toth/react-hook-local-web-storage.git",
   "license": "LGPL",
   "author": "Bence A. Toth <tothab@gmail.com>",
+  "contributors": [
+    "Aleksander Nowak (https://github.com/vyrwu)"
+  ],
   "main": "dist/index.js",
+  "dependencies": {
```

Comment:
Not sure why we need these dependencies

Reply:
changed Babel deps to --dev-only in 5e75fee394bee600d0207f1ec14506d8ff7b4b39

Babel output only adds semicolons to the code. Maybe ditch?

Reply:
Removed babel in ec7f4efd92d7e3cfeeba05e9cd0e8a4ea21b7622